openmeetings-service/src/main/java/org/apache/openmeetings/service/calendar/caldav/IcalUtils.java

```diff
@@ -217,7 +218,9 @@ private Appointment addVEventPropertiestoAppointment(Appointment a, CalendarComp
 			}
 		}
 
-		List<MeetingMember> attList = a.getMeetingMembers() == null ? new ArrayList<>() : a.getMeetingMembers();
+		HashSet<MeetingMember> attList = a.getMeetingMembers() == null ? new HashSet<>()
```

Comment:
I would change it to be `Set<MeetingMember> attList` unless you are using methods specific to `HashSet` later